tst/map/TestMap.kt

```diff
@@ -0,0 +1,31 @@
+package map
+
+import org.junit.Assert
+import org.junit.Test
+
+class TestMap {
+
+    @Test
+    fun givesOutTheVicinity() {
```

Comment:
Tests testing trivial stuff imo are not worth it. It also fixes the current definition of 'vicinity', which makes it very fragile.

If I'd really wanted to unit test smth like this, I'd decouple it, ie had a specific generator class that would produce coordinates for all points on the same axes. And you could unit test that separately, isolated from your domain of Nodes, Map, etc.
And then I'd have a VisualStimuli producer, which would take an abstract coordinates generator and a map, and would extract the respective nodes from the map and return it.
Now , the unit test is not fragile, since the generator's definition is permanently fixed, but instead we have an over engineering problem.